cachito/web/content_manifest.py

```diff
@@ -31,13 +37,14 @@
 class ContentManifest:
     """A content manifest associated with a Cachito request."""
 
-    def __init__(self, request):
+    def __init__(self, request, packages):
         """
         Initialize ContentManifest.
 
         :param Request request: the request to generate a ContentManifest for
         """
         self.request = request
+        self.packages = [Package.from_json(package) for package in packages]
```

Comment:
Furthermore, we can make refactor in the `PackagesData` to ensure every loaded package and dependency from the `packages.json` is a Package object. This is out of this PR's scope. Just share my thought.

Reply:
+1 for that, but it should not be the `content_manifest.Package` class, that one should be for content manifests only. We could have a separate `packages_data.Package` class and then the content_manifest one would just be a wrapper around it that adds purl methods.

Reply:
How about make a general `Package` class? And rename the `content_manifest.Package` to another name to reflect the content manifest generation directly? :)

Reply:
Yeah, in my mind the `packages_data.Package` would be the general one. What would you suggest for the new name, `ContentManifestPackage` as it was called before? Or something else?

Reply:
Hopefully, this is helpful to express my idea.

![cachito-content-manifests](https://user-images.githubusercontent.com/919706/123942562-d6312480-d9cd-11eb-9ae7-d34e0eee5df1.png)